ports/azure-security-keyvault-common-cpp/portfile.cmake

```
@@ -0,0 +1,19 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO Azure/azure-sdk-for-cpp
+    REF azure-security-keyvault-common_4.0.0-beta.1
+    SHA512 7364bf775bbf6115bf54ee0c2df8b2bae35ea31c669fcbc358edd00a948f2c73926fbf88282ff5442f40ead9cdfc78661a08e4f86002ac2eac5e78ddd13eb33d

```

```suggestion
    SHA512 7364bf775bbf6115bf54ee0c2df8b2bae35ea31c669fcbc358edd00a948f2c73926fbf88282ff5442f40ead9cdfc78661a08e4f86002ac2eac5e78ddd13eb33d
    HEAD_REF master
```
Please see the reference here https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/vcpkg_from_github.md#notes